docs/self-managed/components/orchestration-cluster/core-settings/configuration/logging.md

```diff
@@ -0,0 +1,67 @@
+---
+id: logging
+title: Logging
```

Comment:
The [Zeebe section on logging](https://preview.docs.camunda.cloud/pr-6414/docs/self-managed/zeebe-deployment/configuration/logging/) describes a few more things. Let's try to port them all (unless you decided deliberately against it?!).